CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,132 @@
+# Contributor Covenant Code of Conduct
```

Comment:
I think we should discuss this in the team and see if it is compatible with the project.

Reply:
This PR is a proposition for these Open Source project base files ;)